projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -207,10 +212,17 @@ public function get_item_schema() {
 	 * @return array Prepared menu item.
 	 */
 	private function prepare_menu_item( array $menu_item ) {
+		// Exclude unauthorized menu items.
 		if ( ! current_user_can( $menu_item[1] ) ) {
 			return array();
 		}
 
+		// Exclude hidden menu items.
+		if ( false !== strpos( $menu_item[4], 'hide-if-js' ) ) {
```

Comment:
I am not sure if using `hide-if-js` does make sense in the menu context. Could we consider changing it to `hide-from-menu`? We will need to pass this guideline to Woo too.

Reply:
Is `hide-from-menu` a CSS class defined in WP Core? The reason why WC uses `hide-if-js` is because it's a class maintained in WP Core: https://core.trac.wordpress.org/browser/trunk/src/wp-admin/css/common.css#L168